### DISCUSSION TITLE
General congestion report

### DISCUSSION THREAD

**Discussion by lustefan97** [The-OpenROAD-Project/OpenROAD#4305] [Q&A] [answered]
Hello,

Is it possible to write and obtain a congestion report at all time, even when there is no violation ? I am aware of the possibility to have a report written when the wire congestion is too high, but I don't see a way to get the heat map data in a text file for example.

Is there a way to do such a thing ?

**Comment by maliberty** [comment, marked as answer]
In https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/gui/README.md#heat-maps see gui::dump_heatmap
